Framework/PythonInterface/mantid/plots/helperfunctions.py

```diff
@@ -31,7 +32,7 @@
 def validate_args(*args):
     return len(args) > 0 and (isinstance(args[0], EventWorkspace) or
                               isinstance(args[0], Workspace2D) or
-                              isinstance(args[0], MDHistoWorkspace))
+                              isinstance(args[0], MultipleExperimentInfos))
```

Comment:
You need to exclude MDEventWorkspaces if you don't specify a log. Otherwise bad things will happen